### DISCUSSION TITLE
Suggested way to override default-theme SCSS file?

### DISCUSSION THREAD

**Discussion by selected-pixel-jameson** [vuepress/core#155] [General]
I'm extending the default theme, but will have lots of Global CSS that I need to implement along with it. Some of it will override the default-themes CSS files. Some of it will be new additional CSS.

Would I just create CSS / SCSS files within the theme folder and reference those in the Layout.vue file that is extending the default-theme?

**Comment by meteorlxy** [comment]
Styles in `Layout.vue` is OK if you only have a single layout.

If you want your global styles to be implemented in other layouts, consider importing it in `clientAppEnhance`.
